app/src/main/java/cm/aptoide/pt/home/BottomNavigationView.kt

```diff
@@ -202,12 +203,12 @@ private fun NavigationGraph(
     settingsScreen(
       navigate = navController::navigate,
       navigateBack = navController::popBackStack,
-      showSnack = showSnack
+      showSnack = showSnack,
+      versionName = BuildConfig.VERSION_NAME
```

Comment:
No need to overcomplicate this. Lets use it directly in place for now.

Reply:
Can´t use it directly because the `VersionName` field in `BuildConfig` is only available in the app module, not in settings-module